app/main.py

```diff
@@ -1,3 +1,65 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance) -> Distance:
+        if isinstance(other, type(self)):
+            return Distance(self.km + other.km)
+        elif isinstance(other, (int, float)):
+            return Distance(self.km + other)
+
+    def __iadd__(self, other: Distance) -> int:
+        if isinstance(other, type(self)):
+            self.km = self.km + other.km
+            return self
+        elif isinstance(other, (int, float)):
+            self.km = self.km + other
+            return self
+
+    def __mul__(self, other: Distance) -> int:
+        if isinstance(other, (int, float)):
+            return Distance(self.km * other)
+
+    def __truediv__(self, other: Distance) -> int:
+        if isinstance(other, (int, float)):
+            return Distance(round(self.km / other, 2))
```

Comment:
In these two methods `other` can be only int / float, not Distance
```suggestion
    def __mul__(self, other: int | float) -> int:
        if isinstance(other, (int, float)):
            return Distance(self.km * other)

    def __truediv__(self, other: int | float) -> int:
        if isinstance(other, (int, float)):
            return Distance(round(self.km / other, 2))
```

Reply:
do it